docs/topics/mta-6-installing-web-console-on-openshift.adoc

```diff
@@ -161,6 +161,34 @@ spec:
 ** *Password*: Passw0rd!
 . When prompted, create a new password.
 
+[id="installing-mta-operator-in-disconnected-environment_{context}"]
+== Installing the {ProductName} Operator in a disconnected {ocp-short} environment
+
```

Comment:
This really should be a separate module, and probably a different section as we have for MTC https://access.redhat.com/documentation/en-us/openshift_container_platform/4.8/html-single/migration_toolkit_for_containers/index#installing-mtc-restricted

It is a separate procedure, so I suggest changing line 169 to:

.Procedure

* Configure ...

Reply:
@RichardHoch 
Re line 169: It's not really a procedure, it's just an addition to the procedure referenced in line 167.

Reply:
@RichardHoch  Tweaked, thank you